PowerShell/ScubaGear/Modules/CreateReport/CreateReport.psm1

```diff
@@ -183,6 +183,8 @@ function New-Report {
     $ReportHTMLPath = Join-Path -Path $ReporterPath -ChildPath "IndividualReport"
     $ReportHTML = (Get-Content $(Join-Path -Path $ReportHTMLPath -ChildPath "IndividualReport.html")) -Join "`n"
     $ReportHTML = $ReportHTML.Replace("{TITLE}", $Title)
+    $BaselineURL = "<a href=`"$($ScubaGitHubUrl)/blob/v$($SettingsExport.module_version)/PowerShell/ScubaGear/baselines/$($BaselineName.ToLower()).md`" target=`"_blank`"><h3 style=`"width: 100px;`">Baseline Documents</h3></a>"
```

Comment:
This line is problematic because the baseline folder has been moved as part of the power shell refactor. The previous versions are in a different location.